src/components/admin/input-helper-text.tsx

```diff
@@ -0,0 +1,13 @@
+import { ReactNode } from "react";
+
+export function InputHelperText({
+  error,
+  helperText,
+}: {
+  error?: string;
+  helperText?: ReactNode;
+}) {
+  if (!error && !helperText) return null;
+
+  return <p className="text-sm text-destructive mt-1">{error || helperText}</p>;
```

Comment:
Not sure we want to do this as we have the `<Translate>` component but we usually check that props that may contain a string are actually strings and translate them.